src/main/java/uk/gov/hmcts/ccd/domain/model/std/MessageInformation.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.ccd.domain.model.std;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@SuppressWarnings("checkstyle:SummaryJavadoc") // Javadoc predates checkstyle implementation in module
+public class MessageInformation {
+    @JsonIgnore
+    private Long id;
+    @JsonProperty("user_id")
+    private String userId;
+    @JsonProperty("jurisdiction_id")
+    private String jurisdictionId;
+    @JsonProperty("case_type_id")
+    private String caseTypeId;
+    @JsonProperty("case_id")
+    private String caseId;
+    @JsonProperty("event_timestamp")
+    private LocalDateTime eventTimestamp;
+    @JsonProperty("event_instance_id")
+    private Long eventInstanceId;
+    @JsonProperty("event_id")
+    private String eventId;
+    @JsonProperty("previous_state_id")
+    private String previousStateId;
+    @JsonProperty("new_state_id")
+    private String newStateId;
+    @JsonProperty("additional_data")
+    @ApiModelProperty("A JSON object (for future use) that contains additional case fields that have been configured "
+        + "to be included in the event information that is published.")
+    private Map<String, JsonNode> data;
```

Comment:
FYI for the upcoming phase 2 the `additionalData` field will be changed to contain 2 fields: `data` and `definition`. See: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1460561625#WorkAllocationPhase2ScopeofDelivery-NewMessageQueueCandidatesDatabaseStructure
Mentioning this in case you prefer to update to the new structure as part of the PR or later on phase 2. As you prefer

Reply:
done 👍